.github/workflows/build-iso.yml

```
@@ -33,6 +33,12 @@
     steps:
     - uses: actions/checkout@v4
 
+    - name: Get ISO Build Date
+      id: iso_build_date
+      run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:2:32: Double quote to prevent globbing and word splitting

(shellcheck)